docs/client/install-kn.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing the kn CLI"
+weight: 10
+type: "docs"
+aliases:
+  - /docs/install/install-kn
+  - /docs/client/connecting-kn-to-your-cluster
+---
+
+This guide provides details about how you can install the Knative `kn` client (CLI) using various methods.
+
+## Install kn using an executable binary
+
+You can install `kn` by downloading the executable binary for your system and placing it in the system path.
+
+Links to the latest stable executable binary releases are available on the <a href="https://github.com/knative/client/releases" target="_blank">`kn` release page</a>.
+
+
+### Install kn using nightly executable binary
+
+Nightly executable binaries are available for users who want to install the latest pre-release build of `kn`.
+
+**WARNING:** Nightly executable binaries include features which may not be included in the latest Knative release and are not considered to be stable.
+
+Links to the latest nightly executable binaries are available here:
+
+- <a href="https://storage.googleapis.com/knative-nightly/client/latest/kn-darwin-amd64" target="_blank">macOS</a>
+- <a href="https://storage.googleapis.com/knative-nightly/client/latest/kn-linux-amd64" target="_blank">Linux</a>
+- <a href="https://storage.googleapis.com/knative-nightly/client/latest/kn-windows-amd64.exe" target="_blank">Windows</a>
+
+## Install kn using Go
```

Comment:
"Build kn yourself" ???

Reply:
Saying build it yourself doesn't give a clear idea of what's actually required to do that, I think "using Go" is better tbh